js/index.js

```diff
@@ -39,4 +39,37 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+// Assign data
+
+// Nav items
+Array.from(document.getElementsByTagName("nav")[0].childNodes)
+    .filter(e => e.tagName === 'A')
```

Comment:
`Array.from()` works for creating an array for DOM elements. there is also another way I just recently learned that I think is cool by using the spread operator.
`[...document.getelementsByTagName('nav')[0].childNodes]` and then used what ever methods here.